egs/ljspeech/tts1/local/setup_knowledge_dist.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Copyright 2019 Tomoki Hayashi
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+# Setup json files for knowledge distillation training in FastSpeech
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+
+# basic setting
+stage=0
+stop_stage=100
+backend=pytorch
+nj=32
+dumpdir=dump
+verbose=1
+
+# teacher model related
+teacher_model_path=
+decode_config=conf/decode_for_knowledge_dist.yaml
+
+# data related
+trans_type=phn
+dict=
+train_set=phn_train_no_dev
+dev_set=phn_dev
+
+# filtering related
+do_filtering=false
+focus_rate_thres=0.65
+
+. utils/parse_options.sh || exit 1;
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# check arguments
+if [ -z "${teacher_model_path}" ]; then
+    echo "you must set teacher_model_path." 2>&1
+    exit 1;
+fi
+if [ -z "${dict}" ]; then
+    echo "you must set dict." 2>&1
+    exit 1;
+fi
+
+expdir=$(dirname "$(dirname "${teacher_model_path}")")
+outdir=${expdir}/outputs_$(basename ${teacher_model_path})_$(basename ${decode_config%.*})
+if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
+    echo "stage 0: Decoding for knowledge distillation"
+    pids=() # initialize pids
+    for name in ${dev_set} ${train_set}; do
+    (
+        [ ! -e "${outdir}/${name}" ] && mkdir -p "${outdir}/${name}"
+        cp ${dumpdir}/${name}/data.json "${outdir}/${name}"
+        splitjson.py --parts ${nj} "${outdir}/${name}/data.json"
+        # shellcheck disable=SC2154
+        ${train_cmd} JOB=1:${nj} "${outdir}/${name}/log/decode.JOB.log" \
+            tts_decode.py \
+                --backend ${backend} \
+                --ngpu 0 \
```

Comment:
I added npu option.
But keep cpu decoding as default because in the taco2 case it requires around 30 minutes with `nj=32`. That is acceptable, I think.
(But Transformer is too slow. Maybe even if we use GPU.)